docs/layers/arc-layer.md

```diff
@@ -6,10 +6,6 @@
 
 # ArcLayer
 
-The Arc Layer renders raised arcs joining pairs of source and target points,
```

Comment:
Just to clarify, when I commented about "source link" should be separated from the paragraph, I meant the link to the original source code in the repo.
I.e this piece:
Source](https://github.com/uber/deck.gl/tree/master/src/layers/core/arc-layer)